src/main/java/control/ToolViewController.java

```diff
@@ -36,10 +34,10 @@ public ToolViewController(ControllerManager controllerManager) {
      * the event handlers.
      */
     private void initializeTools() {
-        cameraBlockCreationTool = new ToolButton("Add camerashot",
+        new ToolButton("Add camerashot",
```

Comment:
Fair point, I was just checking. Deleting them seems like the right thing to do.